src/styles/style.scss

```diff
@@ -438,8 +434,16 @@ input {
   letter-spacing: 1px;
   border-radius: 5px;
 }
+input.non-valid {
+  margin: 10px 0;
+  border: 2px solid $start-color;
+  padding: 5px 10px;
+  font-size: 14px;
+  letter-spacing: 1px;
+  border-radius: 5px;
+}
```

Comment:
If you only change the border color, then don't copy the whole code, but instead add only the background color.
Like in row 440, add
`&.non-valid {
  border: 2px solid $start-color;
}`